src/main/java/com/openshift/internal/restclient/model/BuildConfig.java

```diff
@@ -142,6 +200,14 @@ public void setOutput(DockerImageURI imageUri){
 		return null;
 	}
 
+	private void storeEnvironmentVars(final String [] key, ModelNode root, Map<String, String> envs){
```

Comment:
Right, was wondering about that. I ended up just implementing it as how 'loadEnvrionmentVars' was implemented :)